pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -47,21 +46,21 @@ buildPythonPackage rec {
         darwin.cpu = {
           py2 = {
             url = tfurl "mac" "cpu" "py2-none-any" ;
-            sha256 = "1fgf26lw0liqxc9pywc8y2mj8l1mv48nhkav0pag9vavdacb9mqr";
+            sha256 = "0nkymqbqjx8rsmc8vkc26cfsg4hpr6lj9zrwhjnfizvkzbbsh5z4";
           };
           py3 = {
             url = tfurl "mac" "cpu" "py3-none-any" ;
-            sha256 = "0z5p1fra7bih0vqn618i2w3vyy8d1rkc72k7bmjq0rw8msl717ia";
+            sha256 = "1rj4m817w3lajnb1lgn3bwfwwk3qwvypyx11dim1ybakbmsc1j20";
           };
         };
         linux-x86_64.cpu = {
           py2 = {
             url = tfurl "linux" "cpu" "cp27-none-linux_x86_64";
-            sha256 = "0ld3hqx3idxk0zcrvn3p9yqnmx09zsj3mw66jlfw6fkv5hznx8j2";
+            sha256 = "09pcyx0yfil4dm6cij8n3907pfgva07a38avrbai4qk5h6hxm8w9";
           };
-          py35 = {
-            url = tfurl "linux" "cpu" "cp35-cp35m-linux_x86_64";
-            sha256 = "0ahz9222rzqrk43lb9w4m351klkm6mlnnvw8xfqip28vbmymw90b";
+          py3 = {
+            url = tfurl "linux" "cpu" "cp36-cp36m-linux_x86_64";
+            sha256 = "1qm8lm2f6bf9d462ybgwrz0dn9i6cnisgwdvyq9ssmy2f1gp8hxk";
           };
```

Comment:
@jyp The problem is for example here, notice `py3` and `py36` are both in your result. Git still managed to merge it without conflicts ;)